test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -10541,6 +10568,9 @@ def revert_to_defaults(self):
         """
         self.logger.info(self.logprefix +
                          "reverting configuration to defaults")
+        self.server.manager(MGR_CMD_DELETE,
```

Comment:
Above call is required to to clean the scheduler created by testcase(s) . No changes required in PTL framework to delete sched.